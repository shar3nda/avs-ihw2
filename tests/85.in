o<KD.5KB'ntx#?{oXCH<dgc.`iDQ25UZ=CQsii:RjU#GE;-%Y:0]s)Y1b]ajQ5DYp'wqdP^gQvsRbw11