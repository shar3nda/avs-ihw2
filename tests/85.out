Lowercase: 485, uppercase: 447